lib/spack/spack/cmd/config.py

```diff
@@ -90,6 +91,14 @@ def setup_parser(subparser: argparse.ArgumentParser) -> None:
         choices=("all", "env", "include", "internal", "path"),
         help="list only scopes of the specified type(s)\n\noptions: %(choices)s",
     )
+    scopes_parser.add_argument(
```

Comment:
(nitpick) Maybe a header would be nice for readability:
```console
% spack config scopes -vv
command_line     internal      active   
spack            path          active   
user             include,path  active   
site             include,path  active   
defaults         path          active   
defaults:darwin  include,path  active   
defaults:base    include,path  active   
_builtin         internal      active   
```

Reply:
it's there -- the headers only appear when there is more than one column.